packages/kit/src/views/Onboardingv2/pages/SelectPrivateKeyNetwork.tsx

```
@@ -50,9 +50,10 @@ import {
 } from '@onekeyhq/shared/src/engine/engineConsts';
 import { ETranslations } from '@onekeyhq/shared/src/locale';
 import { defaultLogger } from '@onekeyhq/shared/src/logger/logger';
-import type {
-  EOnboardingPagesV2,
-  IOnboardingParamListV2,
+import {
+  ERootRoutes,
+  type EOnboardingPagesV2,
```

ESLint

[error] Member 'EOnboardingPagesV2' of the import declaration should be sorted alphabetically.

(sort-imports)